File: 344. Reverse String/C++/Reverse String.cpp
class Solution {
public:
    string reverseString(string s) {
        for (int i = 0, j = s.length() - 1; i < j; i++, j--) {
            char temp = s[i];
            s[i] = s[j];
            s[j] = temp;
        }
        return s;
    }
};

File: 344. Reverse String/Java/Reverse String.java
public class Solution {
    public String reverseString(String s) {
        char[] charArray = s.toCharArray();
        for (int i = 0, j = charArray.length - 1; i < j; i++, j--) {
            char temp = charArray[i];
            charArray[i] = charArray[j];
            charArray[j] = temp;
        }
        return new String(charArray);
    }
}